### PR TITLE
[ADD][8.0] sale_service_fleet: New module.

### PR DESCRIPTION
**Pull request by sergio-teruel**
This module was written to extends the functionality of sale service project and allow you to assign a vehicle to a sale order, so when the project is created the vehicle is assigned to it

This module depends in https://github.com/OCA/sale-workflow/pull/226

### CODE REVIEW

**Comment by rafaelbn** [comment]
Please @sergio-incaser rebase and check travis, we will check runbot after https://github.com/OCA/sale-workflow/pull/226 is merged, thanks!

**Comment by rafaelbn** [comment]
Please rebase, thanks

**Comment by carlosdauden** [comment]
@rafaelbn rebased

**Comment by rafaelbn** [comment]
I have tested this module in local and is in customer's production environment. functionally is :+1: 

could you say me is Travis errors is the same reason as https://github.com/OCA/sale-workflow/pull/226#issuecomment-187073672?

Thanks

**Comment by carlosdauden** [comment]
Yes @rafaelbn. Travis fails by the same reason  as https://github.com/OCA/sale-workflow/pull/226#issuecomment-187073672 and a new not related test error is showed now:

`2016-02-23 14:51:38,335 23832 ERROR openerp_test openerp.addons.sale_order_lot_selection.tests.test_sale_order_lot_selection: File "/home/travis/odoo-8.0/addons/stock/wizard/stock_transfer_details.py", line 79, in do_detailed_transfer 2016-02-23 14:51:38,335 23832 ERROR openerp_test openerp.addons.sale_order_lot_selection.tests.test_sale_order_lot_selection: raise Warning(_('You cannot transfer a picking in state \'%s\'.') % self.picking_id.state) 2016-02-23 14:51:38,335 23832 ERROR openerp_test openerp.addons.sale_order_lot_selection.tests.test_sale_order_lot_selection: openerp.exceptions.Warning: You cannot transfer a picking in state 'False'.`

File: sale_service_fleet/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+Sale Service Fleet
+==================
+
+This module was written to extends the functionality of sale service project
```
Comment (yajo) [inline sale_service_fleet/README.rst]: "extend"

File: sale_service_fleet/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/8.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
```
Comment (yajo) [inline sale_service_fleet/README.rst]: Please remove empty sections.

File: sale_service_fleet/i18n/sale_service_fleet.pot [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Translation of Odoo Server.
```
Comment (yajo) [inline sale_service_fleet/i18n/sale_service_fleet.pot]: .pot files are not needed nowadays.

File: sale_service_fleet/report/project_report.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+
+class ReportProjectTaskUser(models.Model):
+    _inherit = 'report.project.task.user'
+
+    vehicle_id = fields.Many2one(
+        comodel_name='fleet.vehicle', string='Vehicle', readonly=True)
+
+    def init(self, cr):
+        tools.sql.drop_view_if_exists(cr, 'report_project_task_user')
```
Comment (yajo) [inline sale_service_fleet/report/project_report.py]: Please add a docstring here explaining what does this method do.

**Comment by yajo** [comment]
Code seems quite good, wating for runbot to try it.

**Comment by sergio-teruel** [comment]
@Yajo @rafaelbn Changes done and waiting for travis...

File: sale_service_fleet/README.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+This module was written to extend the functionality of sale service project
+and allow you to assign a vehicle to a sale order, so when the project is
+created the vehicle is assigned to it.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a vehicle in fleet menu and assign it to a sale order.
```
Comment (yajo) [inline sale_service_fleet/README.rst]: I see in the code many more functionalities. Could you explain yourself better please? Just to know what am I supposed to test in the runbot.

**Comment by rafaelbn** [comment]
Travis still fails :cry: , please take a review in explaining a little bit more. Thanks!

**Comment by rafaelbn** [comment]
Absolutely great and clean! :+1:

**Comment by yajo** [comment]
:+1:

File: sale_service_fleet/models/fleet.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class FleetVehicle(models.Model):
+    _inherit = "fleet.vehicle"
+
+    project_ids = fields.One2many(
```
Comment (pedrobaeza) [inline sale_service_fleet/models/fleet.py]: Why projects and not analytic accounts?

File: sale_service_fleet/report/project_report.py [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+        # Adds t.vehicle_id to select and group in tasks report view
+        cr.execute("""
```
Comment (pedrobaeza) [inline sale_service_fleet/report/project_report.py]: Please don't replace totally a method, or you kill the inheritance chain. Use a similar technique that I use in https://github.com/OCA/crm/pull/74/files#diff-66b637b1e994d4439cf089b62a5a0ed5R14

**Comment by carlosdauden** [comment]
@pedrobaeza changes done

**Comment by pedrobaeza** [comment]
Please remove pot file and regenerate es.po one (for the new field).

**Comment by carlosdauden** [comment]
@pedrobaeza string changed an translated

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**